apps/marketplace/components/BuyerSpecialist/BuyerSpecialistTimeframesAndBudgetStage.js

```diff
@@ -11,8 +11,10 @@ import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
 
 const contractLengthRequired = v => required(v.contractLength)
 const startDateRequired = v => required(v.startDate)
-const startDateIsValid = v => validDate(v.startDate)
-const startDateIs2DaysInFuture = v => !startDateRequired(v) || !startDateIsValid(v) || dateIs2DaysInFuture(v.startDate)
+const startDateIsValid = v => !startDateRequired(v) || validDate(v.startDate, false)
+const startDateIsInPast = v => !startDateRequired(v) || validDate(v.startDate, true) || !validDate(v.startDate, false)
```

Comment:
To follow the pattern of the other functions, does it make sense to redefine this as the following?

```suggestion
const startDateIsInPast = v => !startDateRequired(v) || !startDateIsValid(v) || validDate(v.startDate, true)
```